meson.build

```
@@ -26,6 +26,7 @@ rt = cc.find_library('rt')
 pipewire = dependency('libpipewire-0.3', version: '>= 0.3.2')
 wayland_client = dependency('wayland-client')
 wayland_protos = dependency('wayland-protocols', version: '>=1.14')
+iniparser = cc.find_library('iniparser')
```

Fails on FreeBSD (and likely other BSDs) as packages are not in the default search path of base compiler in order to avoid tainting build environment. See also https://wiki.freebsd.org/WarnerLosh/UsrLocal#Library_paths

```
$ meson setup _build
[...]
meson.build:29:0: ERROR: C shared or static library 'iniparser' not found

$ pkg info -l iniparser
iniparser-4.1:
        /usr/local/include/dictionary.h
        /usr/local/include/iniparser.h
        /usr/local/lib/libiniparser.a
        /usr/local/lib/libiniparser.so
        /usr/local/lib/libiniparser.so.1
        /usr/local/share/licenses/iniparser-4.1/LICENSE
        /usr/local/share/licenses/iniparser-4.1/MIT
        /usr/local/share/licenses/iniparser-4.1/catalog.mk
```

Maybe try the following
```suggestion
iniparser = cc.find_library('iniparser', dirs: [join_paths(get_option('prefix'), get_option('libdir'))])
```

---

Sorry, I was just looking through PRs that may break recently added FreeBSD build on CI and didn't notice this change was part of another PR. I'll move the comment there shortly.